cypresse2e/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+    "baseUrl": "http://staging.fightpandemics.work/",
```

Comment:
Shouldn't it be `https`? Like this it will be redirected to https but probably better just to have https here.

Reply:
Ok, corrected